packages/blocks/src/page-block/edgeless/services/selection-manager.ts

```diff
@@ -206,7 +206,10 @@ export class EdgelessSelectionManager {
     this._selection.setGroup('edgeless', [this.state, instance]);
   }
 
-  clear() {
+  clear(force = false) {
```

Comment:
Instead of adding new option, can we switch the default behavior to `force: true`?

Reply:
@doouding @Saul-Mirone Can we directly always clear all the selections in edgeless mode when call selection manager clear function?